src/test/java/reposense/parser/PathLocationParserTest.java

```diff
@@ -0,0 +1,32 @@
+package reposense.parser;
+
+import static org.junit.Assert.assertArrayEquals;
+
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+public class PathLocationParserTest {
+
+    @Test
+    public void pathLocationParser_parsePath_success() throws Exception {
```

Comment:
Can you give one test for a failed path? For example - Path doesn't exist, or if it does exist it doesn't contain Git Directory?

Reply:
I can add the following tests:
- Test that `InvalidLocation` exception is thrown if repo directory does not exist
- Test that `InvalidLocation` exception is thrown if the `.git` directory specified by user does not exist.

Note: If the user simply provides the path to a repo directory (not the repo's `.git` directory), then we do not actually validate that the repo directory contains a `.git` directory. We simply catch the exception that is thrown during the cloning step. I have not modified this behaviour; this is the current RepoSense behaviour.